test/e2e/kubelet_deletes.go

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
Yes, I'd rather rename the file to kubelet.go, even if you don't merge it with kubelet_stats.go.